packages/ui-library/src/components/token-picker-button/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from 'styled-components';
+import { colors, baseColors } from '../../theme';
+import { TokenWithIcon } from 'common-types';
+import { ButtonProps, Skeleton, Typography } from '@mui/material';
+import { SPACING } from '../../theme/constants';
+import { FormattedMessage } from 'react-intl';
+import { Button } from '../button';
+import { KeyboardArrowDownIcon } from '../../icons';
+
+const StyledTokenPickerButton = styled(Button)`
+  ${({ theme: { spacing, palette } }) => `
+  display: flex;
+  align-items: center;
+  gap: ${spacing(2)};
+  padding: ${spacing(2)};
+  border: 1px solid ${colors[palette.mode].border.border1};
```

Comment:
shouldn't border be part of the button variant?

Reply:
We have no outline buttons defined in our design, so this can be the standard now :)